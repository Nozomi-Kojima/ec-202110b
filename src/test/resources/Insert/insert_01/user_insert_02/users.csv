name,email,zipcode,address,telephone
テストユーザ,testtest@gmail.com,123-4567,神奈川県相模原市,090-1234-5678