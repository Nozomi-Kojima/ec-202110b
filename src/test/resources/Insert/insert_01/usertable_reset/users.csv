name,email,zipcode,address,telephone