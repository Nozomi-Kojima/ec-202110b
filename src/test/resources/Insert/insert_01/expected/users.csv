name,email,zipcode,address,telephone
テストユーザ,ramen.test@gmail.com,1111111,テスト住所,テスト電話番号
テストユーザ1,testtest@gmail.com,123-4567,神奈川県相模原市,090-1234-5678